src/mame/drivers/pcw.cpp

```diff
@@ -1020,6 +1021,7 @@ void pcw_state::machine_reset()
 	m_printer_headpos = 0x00; // bring printer head to left margin
 	m_printer_shift = 0;
 	m_printer_shift_output = 0;
+	machine().render().first_target()->set_view(0);
```

Comment:
This breaks expectations.  MAME saves/restores selected view automatically, and you've broken that.  It also won't play nicely with external artwork either (the first view will come from the external artwork, so you can't make assumptions about it), it probably breaks the `-view0` etc. command-line options, and if the user configures the system to have an additional screen (e.g. with a terminal) it won't show by default.

Reply:
ok, I just want to get rid of the printer screen on startup, it ain't useful until you print something and de-centers the terminal view